frontend/app/templates/components/mapbox/feature-selector.hbs

```diff
@@ -0,0 +1,9 @@
+<Mapbox::CurrentMouseEvent @mapboxEventName="click" @map={{this.map}} as |mapMouseEvent|>
+  <Mapbox::IntersectingFeatures
+    @map={{this.map}}
+    @point={{mapMouseEvent.point}}
+    @options={{hash layers=(array this.layerId)}} as |intersectingFeatures|
```

Comment:
Minor: We might consider making this more generic by having `feature-selector` accept an array of layerIds, rather than allow one.